pkgs/development/python-modules/soco/default.nix

```diff
@@ -1,29 +1,54 @@
-{ lib, buildPythonPackage, fetchPypi, xmltodict, requests
-, toml
-
-# Test dependencies
-, pytest, pytestcov, coveralls, pylint, flake8, graphviz, mock, sphinx
+{ buildPythonPackage
+, coveralls
+, fetchFromGitHub
+, flake8
+, graphviz
+, lib
+, mock
+, pytestCheckHook
+, requests
+, sphinx
 , sphinx_rtd_theme
+, toml
+, xmltodict
 }:
 
 buildPythonPackage rec {
   pname = "soco";
   version = "0.20";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "929d4fae20b32efc08bb9985798c592aa7268162885541513eddbff0a757418f";
+  src = fetchFromGitHub {
```

Comment:
Add a comment, that PyPi does not contain the necessary test data.

Reply:
Done